services/traction/api/endpoints/routes/v1/tenant/issuer/credentials.py

```diff
@@ -0,0 +1,141 @@
+import logging
+from typing import List
+
+from fastapi import APIRouter, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+from starlette import status
+
+from api.services.v1 import issuer_service
+from api.services.v1.issuer_service import IssueCredentialData
+
+from api.db.models.v1.contact import Contact
+
+from api.endpoints.dependencies.tenant_security import get_from_context
+from api.endpoints.dependencies.db import get_db
+
+
+from api.endpoints.models.credentials import (
+    IssueCredentialProtocolType,
+    CredentialPreview,
+)
+from api.endpoints.models.tenant_workflow import (
+    TenantWorkflowStateType,
+)
+
+from api.endpoints.models.v1.issuer import CredentialsListResponse, CredentialItem
+
+router = APIRouter()
+logger = logging.getLogger(__name__)
+
+
+@router.get("/credentials", response_model=CredentialsListResponse)
+async def get_issued_credentials(
+    state: TenantWorkflowStateType | None = None,
+    workflow_id: str | None = None,
+    cred_issue_id: str | None = None,
+    db: AsyncSession = Depends(get_db),
+) -> List[IssueCredentialData]:
+    # this should take some query params, sorting and paging params...
+    wallet_id = get_from_context("TENANT_WALLET_ID")
+    tenant_id = get_from_context("TENANT_ID")
+
+    data = await issuer_service.get_issued_credentials(
+        db, tenant_id, wallet_id, workflow_id, cred_issue_id, state
+    )
+    logger.debug(data)
+    resp_data = [
```

Comment:
the service should be returning the API objects

Reply:
That's not as feasible right now because the v0 endpoint uses that service method to (i refactored it that way). 

I could just fully duplicate the implementation then we can get this behaviour.

Reply:
the v1 service can wrap all the v0 stuff for now, that's what I was doing with contacts (create/receive an invitation), so in the future, you'd (theoretically) just change the internals of the service to do only v1 work.